bfrec

```diff
@@ -266,6 +285,15 @@ if [ -n "$bootctl_mode" ] || [ -n "$capsule_mode" ]; then
     fi
 fi
 
+if [ -n "$policy" ]; then
+    if [ "$policy" != single ] && [ "$policy" != dual ]; then
```

Comment:
Suggest use "single" and "dual" to be consistent, Or is there any reason not to use it?

How about 
[ "$policy" != "single" -a "$policy" != "dual" ] ?
Maybe we could use the checkbashisms to see which one is better

Reply:
> Suggest use "single" and "dual" to be consistent, Or is there any reason not to use it?

Done.

> How about
> [ "$policy" != "single" -a "$policy" != "dual" ] ?
> Maybe we could use the checkbashisms to see which one is better

Checked both with checkbashisms, both are fine. I picked
```bash
if [ "$policy" != single ] && [ "$policy" != dual ]; then
```
to match the above line...
```bash
if [ -n "$bootctl_mode" ] || [ -n "$capsule_mode" ]; then
```